platform/darwin/src/http_file_source.mm

```diff
@@ -371,6 +371,8 @@ BOOL isValidMapboxEndpoint(NSURL *url) {
                             std::make_unique<Error>(Error::Reason::Other, std::string{ "HTTP status code " } +
                                                                               std::to_string(responseCode));
                     }
+                } else if ([url isFileURL]) {
+                    response.data = std::make_shared<std::string>((const char *)[data bytes], [data length]);
```

Comment:
Is this change required?

Reply:
I temporarily cherry-picked the changes in #16428 so that the tests in mapbox/mapbox-gl-native-ios#189 would build. Once #16428 lands, I’ll remove the cherry-pick.